jdatetime/jalali.py

```diff
@@ -1,6 +1,28 @@
-# dervied from http://farsitools.sf.net
-# Copyright (C) 2003-2011  Parspooyesh Fanavar (http://parspooyesh.com/)
-# see LICENSE.txt
+# This file is part of:
```

Comment:
Maybe we can start the file by clarifying this work is based on the C implementation (http://www.farsiweb.info/jalali/jalali.c).

IMHO we can start:

````
The algorithms to convert between Jalali and Gregorian dates, are based on
the algorithms implemented by the C library (http://www.farsiweb.info/jalali/jalali.c).
Which is part of:
    ...
````